src/config/index.js

```diff
@@ -9,6 +9,16 @@ const config = {
     OPTIMIZATION_TIME: process.env.OPTIMIZATION_TIME || 10 * 60 * 1000, // 10 minutes
   },
 
+  db: {
+    user: process.env.DB_USER || fatalError('FATAL: DB_USER is not defined'),
+    host: process.env.DB_HOST || fatalError('FATAL: DB_HOST is not defined'),
+    port: process.env.DB_PORT || fatalError('FATAL: DB_PORT is not defined'),
```

Comment:
Error: fatalError is not a function

Reply:
I fixed it move the fatalError function in to config. Did you mean this?